chapters/ch02.md

```diff
@@ -1,38 +1,32 @@
-# Why Machine Learning Strategy
-
+# How to use this book to help your team 
 ->
 
-
+# Cách sử dụng cuốn sách khi làm việc nhóm
 
 After finishing this book, you will have a deep understanding of how to set technical direction for a machine learning project.
 ->
-
-
+Sau khi đọc xong cuốn sách này, bạn sẽ hiểu sâu hơn về cách lựa chọn hướng giải quyết kỹ thuật cho đề tài machine learning. 
 
 But your teammates might not understand why you’re recommending a particular direction.
 ->
-
-
+Nhưng có thể cộng sự của bạn không hiểu tại sao bạn lại chọn hướng đi như vậy. 
 
 Perhaps you want your team to define a single-number evaluation metric, but they aren’t convinced. How do you persuade them?
 ->
+Có thể bạn muốn cả đội xác định và dùng phép đo một chỉ số, nhưng liệu tất cả thành viên đều đồng tình với quyết định đó. Vậy bạn sẽ làm gì để thuyết phục họ?
 
 
 
 That’s why I made the chapters short: So that you can print them out and get your teammates to read just the 1-2 pages you need them to know.
 ->
-
-
+Đó là lý do tại sao tôi chủ tâm viết những chương rất ngắn. Bạn có thể dễ dàng thuyết phục quý đồng nghiệp bằng cách chia sẻ bản in của chương liên quan chỉ từ 1 đến 2 trang. 
```

Comment:
Sửa lại một xíu nữa thôi '..chia sẻ 1-2 trang bản in của chương liên quan.

Reply:
update